DuckDuckGoTests/FireButtonReferenceTests.swift

```diff
@@ -44,6 +44,54 @@ final class FireButtonReferenceTests: XCTestCase {
         return url.host!
     }
 
+    func testClearData() async {
+        let preservedLogins = PreserveLogins.shared
+        preservedLogins.clearAll()
+        
+        for site in testData.fireButtonFireproofing.fireproofedSites {
+            let sanitizedSite = sanitizedSite(site)
+            os_log("Adding %s to fireproofed sites", sanitizedSite)
+            preservedLogins.addToAllowed(domain: sanitizedSite)
+        }
+        
+        let referenceTests = testData.fireButtonFireproofing.tests.filter {
+            $0.exceptPlatforms.contains("ios-browser") == false
+        }
+            
+        let cookieStorage = CookieStorage()
+        let idManager = DataStoreIdManager()
+        for test in referenceTests {
+            guard let cookie = cookie(for: test) else {
```

Comment:
I find it better to use `try XCTUnwrap` with a throwing test function in such cases. It gives you automatic error message :).

Reply:
Ah very nice, will use that.